src/nbl/video/IGPUCommandBuffer.cpp

```diff
@@ -1256,7 +1584,10 @@ bool IGPUCommandBuffer::drawIndirectCount(const asset::SBufferBinding<const IGPU
         return false;
 
     if (!m_cmdpool->m_commandListPool.emplace<IGPUCommandPool::CDrawIndirectCountCmd>(m_commandList, core::smart_refctd_ptr<const IGPUBuffer>(indirectBinding.buffer), core::smart_refctd_ptr<const IGPUBuffer>(countBinding.buffer)))
-        return false;
+    {
+        NBL_LOG_ERROR("out of host memory!");
+        return true;
+    }
```

Comment:
why are you now returning a different value?

Please do some eyeballing of all your changes and make sure you don't change the return values

Reply:
> Please do some eyeballing of all your changes and make sure you don't change the return values
done